src/lint/validators.nim

```diff
@@ -0,0 +1,54 @@
+import std/json
+import ".."/helpers
+
+template checkObject*(key: string) =
+  if data.kind != JObject:
+    writeError("Not an object: " & key, path)
+    return false
+  result = true
+
+template checkStringKey*(key: string, isRequired: bool) =
```

Comment:
Is was wondering why this is named `checkStringKey` and not `checkString`, considering the other templates' names.

Reply:
Good point - changed to `checkString`.